Jenkinsfile.properties

```diff
@@ -19,7 +19,7 @@ properties(
 )
 
 // Default if Account ID is not provided
-def awsAccountsOnMerge = ['development-aws', 'integration-aws', 'staging-aws', 'production-aws' , 'engops-aws', 'southwest-aws', 'plcint-aws', 'plcprod-aws']
+def awsAccountsOnMerge = ['development-aws', 'integration-aws', 'staging-aws', 'production-aws' , 'engops-aws', 'southwest-aws', 'plcint-aws', 'plcprod-aws', 'cit-staging-aws']
```

Comment:
@dsayan154 Can we follow the same syntax as other accounts? Also  cit integration and production accounts names start with `plc` so either we should create staging with `plcstating-aws` or change the other account names also to maintain the consistency. 
Right now it feels like plc and cit both are different suits.

Reply:
We won't be using `plc` any longer. We have plans to be moving the rest of CIT accounts to the `cit-<env>` naming format

Reply:
We are using `cit-staging` in terraform-aws-global repository. So I think this should be good.